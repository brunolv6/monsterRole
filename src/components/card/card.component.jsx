import React from 'react';

import './card.style.css';

export const Card = ({ id, name }) => (
    <div className="card">
        <img alt='monsters' src={`https://robohash.org/${id}?set=set2&size=180x180`}/>
        <h3>{name}</h3>
    </div>
);
