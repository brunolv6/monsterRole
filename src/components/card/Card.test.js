
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

import React from 'react';
import { Card } from './card.component';

import toJson from 'enzyme-to-json';

configure({adapter: new Adapter()});

it('testing Card Component', () => {
    expect(toJson(shallow(<Card />))).toMatchSnapshot()
})

