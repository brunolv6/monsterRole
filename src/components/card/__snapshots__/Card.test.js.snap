// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Card Component 1`] = `
<div
  className="card"
>
  <img
    alt="monsters"
    src="https://robohash.org/undefined?set=set2&size=180x180"
  />
  <h3 />
</div>
`;
